### ISSUE TITLE
adler32 crashes on inputs > 31 bytes

### ISSUE THREAD

**Issue by dreamflow** [guzba/crunchy#22]
```nim
import crunchy/adler32
let data_bytes : seq[byte] = newSeq[byte](32)
echo adler32(data_bytes)
```
error : `SIGILL: Illegal operation`

my environment :

cpu : AMD Phenom II X4 970
instructions sets : MMX (+), 3DNow! (+), SSE, SSE2, SSE3, SSE4A, x86-64, AMD-V

nimble list -i :
crunchy  [(version: 0.1.9, ... ]
nimsimd  [(version: 1.2.6, ... ]

nim -v :
Nim Compiler Version 2.1.1 [Windows: amd64]
Compiled at 2023-12-24

gcc --version :
gcc (MinGW-W64 x86_64-ucrt-posix-seh, built by Brecht Sanders) 13.2.0

os : Win10pro-64bit 22H2

**Comment by dreamflow** [comment]
https://github.com/guzba/crunchy/blob/ab917f17647cd54d54de0090224b6c32be315930/src/crunchy/adler32.nim#L17-L18
in line 17 : changing SSE3(with only two "S")
to SSSE3(with three "S") solves the problem .

**Comment by guzba** [comment]
This fix is tagged in release 0.1.10
